src/Compilers/CSharp/Portable/Binder/DecisionDagBuilder.cs

```diff
@@ -60,13 +60,15 @@ internal sealed partial class DecisionDagBuilder
         private readonly Conversions _conversions;
         private readonly BindingDiagnosticBag _diagnostics;
         private readonly LabelSymbol _defaultLabel;
+        private readonly bool _considerAlternativeIndexers;
```

Comment:
>_considerAlternativeIndexers [](http://example.com/codeflow?start=30&length=28)

Consider adding a doc comment.

Reply:
I noticed the name is not entirely true - we do consider alternative indexers, we just avoid synthesizing tests. Will try something more on-point like _forLowering for an easier explanation.